scenarioo-client/app/shared/navigation/branchOrderBy.filter.js

```diff
@@ -40,6 +43,40 @@ function scBranchOrderByFilter() {
             return 1;
         }
 
```

Comment:
yes, Aliases are always in alphabetical order, that was already the case before, and even in the original changes of the origin PR. The order of aliases was never meant to change or to become configurable. 
See next 3 code blocks below this line that you removed, which implemented this special rule for aliases.